SecretSanta/src/SecretSanta.Api/SecretSanta.Api.csproj

```
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+   
+    <Nullable condition ="'$(Nullable)' == ''">enable</Nullable>
```

```suggestion
    <Nullable Condition ="'$(Nullable)' == ''">enable</Nullable>
```
I think this might be causing your Api to not load properly.